lib/Perl/MinimumVersion.pm

```diff
@@ -706,9 +706,10 @@ sub _scheduled_blocks
 }
 
 sub _regex {
+	my $self = shift;
```

Comment:
The indent seems off

Reply:
Made the indent consistent within the `sub` (turned the mix of spaces and tabs into just tabs as that was what most lines were).